app/helpers/sort_helper.rb

```diff
@@ -214,6 +216,12 @@ def sort_clause
     @sort_criteria.to_sql
   end
 
+  # Determines whether the current selected sort criteria
+  # is identical to the default
+  def default_sort_order?
+    @sort_default == @sort_criteria.criteria
```

Comment:
Ugh!
That means it only works, because the caller/client of the `SortHelper` is extra careful. Which is not ideal, because the method is defined on the helper, not on the caller...